PWGEM/PhotonMeson/Tasks/emcalPi0QC.cxx

```diff
@@ -178,12 +182,14 @@ struct Pi0QCTask {
     const o2Axis bcAxis{3501, -0.5, 3500.5};
     const o2Axis energyAxis{makeClusterBinning(), "#it{E} (GeV)"};
 
-    mHistManager.add("events", "events;;#it{count}", o2HistType::kTH1F, {{4, 0.5, 4.5}});
+    mHistManager.add("events", "events;;#it{count}", o2HistType::kTH1F, {{6, 0.5, 6.5}});
     auto heventType = mHistManager.get<TH1>(HIST("events"));
     heventType->GetXaxis()->SetBinLabel(1, "All events");
-    heventType->GetXaxis()->SetBinLabel(2, "One collision in BC");
-    heventType->GetXaxis()->SetBinLabel(3, "Triggered");
-    heventType->GetXaxis()->SetBinLabel(4, "Selected");
+    heventType->GetXaxis()->SetBinLabel(2, "sel8 + readout");
+    heventType->GetXaxis()->SetBinLabel(3, "1+ Contributor");
+    heventType->GetXaxis()->SetBinLabel(4, "z<10cm");
+    heventType->GetXaxis()->SetBinLabel(5, "unique col");
+    heventType->GetXaxis()->SetBinLabel(6, "Non-0 EMCAL cell");
```

Comment:
"Non-0 EMCAL cell" -> "EMCAL cell>0" to be consistent with the other labels

Reply:
Yes, done